echo to stdout when migration-script is running

rdxmb opened the pull request:
fixes https://github.com/nextcloud/docker/issues/327

rdxmb commented:
What's the problem with the Travis Check above?

tilosp commented:
@rdxmb you need to run `update.sh` und commit the changes.

rdxmb commented:
what now?

tilosp commented:
We should fix the ci problems before merging. This currently causes the [`override-cmd`](https://github.com/docker-library/official-images/blob/master/test/tests/override-cmd/run.sh) test to fail. It fails because of the additional version output. And as far as i know this test is required for this to get merged into [docker-library/official-images](https://github.com/docker-library/official-images).

One possible solution would be to not print anything in the startup script if the current version is `0.0.0`

tilosp commented on 12.0/apache/entrypoint.sh:
```diff
@@ -33,7 +33,9 @@ if version_greater "$installed_version" "$image_version"; then
 fi
 
 if version_greater "$image_version" "$installed_version"; then
+    echo -n "Initializing version $image_version ... "
```
the `-n` option, which you use to keep this and `finished.` in one line, does not work if you upgrade from a previous nextcloud version. Because the `occ upgrade` command prints information and is run in betwen this echo and the `finished.` echo.

rdxmb commented:
Just simplified the output and removed the `-n` . We could also run rsync with `-v` - to have an output what is running with `docker logs` . What do you think?

rdxmb commented:
Can somebody help to get this ready? What's the problem with the ci-checks here?

tilosp commented:
@rdxmb once #459 is merged i will rebase this onto master, which should fix the ci.

tilosp commented:
rebased to master

tilosp commented:
Thanks for your contribution!

rdxmb commented:
thanks for merging.
